packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -71,7 +75,10 @@ class YoutubePlayer extends Component {
    */
   componentDidUpdate() {
     const { id, player } = this.state;
-    player.loadVideoById(id, 'default');
+    const { id: propsID } = this.props;
+    const idToUse = id || propsID;
```

Comment:
**Suggestion**: for consistency we can also pull the state ID as `stateID` to completely disambiguate the locals.